src/components/node/NodePreview.vue

```diff
@@ -62,6 +95,11 @@ const props = defineProps({
   }
 })
 
+const litegraphColors = computed(() => {
```

Comment:
`getColorPalette` is not reactive. The `computed` should just be dropped here. Currently the component is recreated so often that it is actually OK to only eval the style once during setup time.